README.md

```diff
@@ -189,6 +189,48 @@ $ image-builder list-images --output=json
 ]
 ```
 
+## Overriding the base repositories
+
+There are various ways to add extra repositories or override the default
+base repositories. Most users will want to use the
+[blueprint systems](https://osbuild.org/docs/user-guide/blueprint-reference/#repositories)
+for this. Repositories that are part of the blueprint will get added
+to the installed image but are not used at build time to install third-party
+packages.
+
+To change repositories during image build time the commandline options
+`--add-repo` and `--override-base-repos-with` can be used. The repositories
```

Comment:
Happy to discuss the name here: `--override-base-repos-with` -  this is just a strawman :)

Reply:
`--force-repo`

Reply:
`--force-repo` is also fine, yes!

Reply:
Thanks, indeed, I misheard during our sync on this I think. Making it symmetric to --extra-repo makes a lot of sense. I updated the PR accordingly. And I like `force-repo`.